index.html

```diff
@@ -40,18 +40,20 @@
 
 </header>
 
-<section class="hero">
+<section class="row hero">
     <div>
         <p class="think">THINK BIG</p>
     </div>
     <div>
         <p class="achieve">ACHIEVE THE IMPOSSIBLE</p>
     </div>
+    <div>
     <button class="btn"> Book Now </button>
```

Comment:
check your indentation here for the button

Reply:
made this change